.azure/pipelines/jobs/default-build.yml

```diff
@@ -155,9 +155,9 @@ jobs:
         displayName: Start background dump collection
     - ${{ if eq(parameters.installNodeJs, 'true') }}:
       - task: NodeTool@0
-        displayName: Install Node 10.x
+        displayName: Install Node 14.x
         inputs:
-          versionSpec: 10.x
+          versionSpec: 14.x
```

Comment:
Should we jump all the way forward to 16.x and catch up w/ 'main'❔ Thoughts @javiercn @BrennanConroy&hellip;

Reply:
I did this to match 6.0: https://github.com/dotnet/aspnetcore/blob/d5d2e4da9e1371533c27d073a09012ff96aeeffb/.azure/pipelines/jobs/default-build.yml#L169-L173